jfrog_platform/manifest.json

```diff
@@ -1,30 +1,36 @@
 {
   "display_name": "JFrog Platform",
   "maintainer": "integrations@jfrog.com",
-  "manifest_version": "1.0.0",
+  "manifest_version": "2.0.0",
   "name": "jfrog_platform",
   "metric_prefix": "jfrog.",
-  "metric_to_check": "",
+  "metric_to_check": "jfrog.artifactory.app_disk_free_bytes",
   "creates_events": false,
-  "short_description": "View and analyze Artifactory events.",
+  "short_description": "View and analyze JFrog Artifactory and Xray Metrics and events.",
   "guid": "2c70552e-b77a-4349-9955-8799b9b57d56",
   "support": "contrib",
   "supported_os": [
     "linux",
     "mac_os",
     "windows"
   ],
-  "public_title": "JFrog Artifactory",
+  "public_title": "JFrog Logs",
```

Comment:
Do you want the integration name to be displayed as this? I'm confused because the integration also emits metrics

Reply:
Thanks for pointing out i have changed the name to JFrog Platform